network/README.md

```diff
@@ -61,7 +61,7 @@ partial membership views, sophisticated failure detectors, or network overlays.
                       +----------------------+--------------------+   +---------------------+
                       |        Peer(s)       |                    |
                       +----------------------+                    |
-                      |                AptosTransport              |
+                      |                AptosTransport             |
```

Comment:
We should really draw a proper graphic SVG diagram of this, using LucidChart. This is not a blocker for now, though.